templates/docs/examples/patterns/tabs/segmented.html

```diff
@@ -0,0 +1,99 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Tabs / Segmented{% endblock %}
+
+{% block standalone_css %}patterns_tabs-segmented{% endblock %}
+
+{% block content %}
+
+<div class="p-tabs-segmented">
+  <div class="p-tabs-segmented__list" role="tablist" aria-label="Juju technology">
+    <button
+      class="p-tabs-segmented__button has-icon"
+      role="tab" 
+      aria-selected="true" 
+      aria-controls="olm-tab" 
+      id="olm"
+    >
+      <i class="p-icon--information"></i>
+      <span>Charmed Operator Lifecycle Manager</span>
+    </button>
+    <button
+      class="p-tabs-segmented__button"
+      role="tab"
+      aria-selected="false"
+      aria-controls="operator-tab"
+      id="operator" 
+      tabindex="-1"
+    >Charmed Operator SDK</button>
+    <button
+      class="p-tabs-segmented__button"
+      role="tab"
+      aria-selected="false"
+      aria-controls="charmhub-tab"
+      id="charmhub"
+      tabindex="-1"
+    >Charmhub</button>
+  </div>
+
+  <div tabindex="0" role="tabpanel" id="olm-tab" aria-labelledby="olm">
```

Comment:
Are these panels part of the component, do they need to be inside the same wrapper?
In real use cases any panel controlled by segmented buttons will probably in different part of the app screen.

Reply:
No, they don't need to be, I'll update